CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+We welcome contributions from the community. Please read the following guidelines carefully to
+maximize the chances of your PR being merged.
+
+# Communication
+
+* Before starting work on a major feature, please reach out to us via GitHub, Slack,
+  email, etc. We will make sure no one else is already working on it and ask you to open a
+  GitHub issue.
+* Small patches and bug fixes don't need prior communication.
+
+# Coding style
+
+* Coding style mirrors [Envoy's policy](https://github.com/envoyproxy/envoy/blob/master/STYLE.md)
+
+# Breaking change policy
+
+Both API and implementation stability are important to Nighthawk. Since the API is consumed by clients beyond Nighthawk, breaking changes to that are prohibited.
+
+# Submitting a PR
+
+* Generally Nighthawk mirrors [Envoy's policy](https://github.com/envoyproxy/envoy/blob/master/CONTRIBUTING.md#pr-review-policy-for-maintainers) with respect to PR submission policy.
+* Any PR that changes user-facing behavior **must** have associated documentation in [docs](docs) as
```

Comment:
(nit) The mentioned docs directory is suspiciously empty, do we have any examples that we can include, or are there some coming shortly? Or is it maybe that we didn't change any behavior yet?

Reply:
Filed https://github.com/envoyproxy/nighthawk/issues/241 and self-assigned